Raven.Database/Storage/Esent/SchemaUpdates/Updates/From54To55.cs

```diff
@@ -0,0 +1,161 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="From54To55.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+using System;
+using Microsoft.Isam.Esent.Interop;
+using Raven.Database.Config;
+using Raven.Database.Impl;
+using Raven.Storage.Esent;
+using Raven.Storage.Esent.SchemaUpdates;
+
+namespace Raven.Database.Storage.Esent.SchemaUpdates.Updates
+{
+    public class From54To55 : ISchemaUpdate
+    {
+        private InMemoryRavenConfiguration configuration;
+
+        public string FromSchemaVersion { get { return "5.4"; } }
+
+        public void Init(IUuidGenerator generator, InMemoryRavenConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
+        public void Update(Session session, JET_DBID dbid, Action<string> output)
+        {
+            using (var tbl = new Table(session, dbid, "lists", OpenTableGrbit.None))
+            {
+                var szIndexName = "+id\0\0";
+                var by_name_and_etag = "+name\0+etag\0\0";
+                var by_name_and_key = "+name\0+key\0\0";
+                var by_name_and_created_at = "+name\0+created_at\0\0";
+
+                var tableid = tbl.JetTableid;
+
+                Api.JetDeleteIndex(session, tbl, "szIndexName");
```

Comment:
Pretty sure this is wrong?

Reply:
I assume you wanted to delete "by_id" index. Although you won't be able to do it because it's primary index. You will get esent error complaining that "clustered index cannot be deleted" or something like that.